M2/Macaulay2/c/README

```diff
@@ -289,7 +289,7 @@ Expressions:
 
 	export a := b				export a variable or function (by declaring it in *.sig)
 	import a				import a variable (used mainly in *.sig files)
-	thread a := b				declare a thread local variable
+	threadLocal a := b			declare a thread local variable
```

Comment:
Can we also change the top level name to `threadLocal` instead of `threadVariable`? It'll be a breaking change, but lots of other things are changing with parallelism anyway. (I should say, I would prefer making them both just be `thread` for brevity, but I particularly dislike `threadVariable`)